src/index.js

```diff
@@ -76,6 +76,25 @@ window.addEventListener("beforeinstallprompt", e => {
   }
 });
 
+// Fetch favorite + public rooms and maerge, sorting by participant count
+async function fetchFeaturedRooms() {
+  const [favoriteRoomsResult, publicRoomsResult] = await Promise.all([
+    authChannel.signedIn
+      ? fetchReticulumAuthenticated(
+          `/api/v1/media/search?source=favorites&type=rooms&user=${store.credentialsAccountId}`
+        )
+      : Promise.resolve({ entries: [] }),
+    fetchReticulumAuthenticated("/api/v1/media/search?source=public_rooms")
+  ]);
+
+  publicRoomsResult.entries.push(...favoriteRoomsResult.entries);
```

Comment:
nit, but I think down the road pushing these here could lead to bugs. i'd create a new list to concat them into with a proper name. (eg, if anyone comes along here and adds code below this point assuming that list contains only public rooms, would introduce bugs)